projects/media-viewer/src/lib/toolbar/toolbar-button-visibility.service.ts

```diff
@@ -0,0 +1,71 @@
+import { BehaviorSubject } from 'rxjs';
+
+export class ToolbarButtonVisibilityService {
+  public readonly subToolbarHidden = new BehaviorSubject(true);
+  public readonly sidebarOpen = new BehaviorSubject(false);
+  public readonly searchBarHidden = new BehaviorSubject(true);
+
+  public showPrint = false;
+  public showDownload = false;
+  public showNavigation = false;
+  public showZoom = false;
+  public showRotate = false;
+  public showPresentationMode = false;
+  public showOpenFile = false;
+  public showBookmark = false;
+  public showHighlight = false;
+  public showSearchBarToggle = false;
+  public showSubToolbarToggle = false;
+  public showSidebarToggle = false;
+
```

Comment:
Feels like the Annotation has been missed?

Reply:
These are the toolbar buttons rather than the toggles at the top of the page to turn features on/off

Reply:
should the draw and highlight mode not be included then?

Reply:
yes, they should. I'm not sure we have added the ability to turn them on/off

Reply:
at present, i had them enabled under the wierd top toolbar Annotate switch, but they should in theory follow the same convention now outlined above